src/Tile/SelectableTileGroup.svelte

```diff
@@ -0,0 +1,52 @@
+<script>
+  /**
+   * Specify the selected tile's
+   * @type {string}
```

Comment:
```suggestion
   * @type {string[]}
```

Reply:
Do you think that should be `any` array? Since the values represent the values you pass to the SelectableTile, and you could give it any type of value?

Reply:
Yes, good point. We should loosen the type here

Reply:
would that be written `@type {any[]}` ?

Reply:
Yes